dhl_sdk/authentication.py

```diff
@@ -33,17 +34,23 @@ class APIKeyAuthentication:
 
     The API Key can be provided as an argument or retrieved from the
     environment variable defined as 'DHL_API_KEY'
-
-    Parameters
-    ----------
-    api_key : str, optional
-        API Key to use to authenticate with the DHL API
     """
 
     def __init__(self, api_key: Optional[str] = None):
+        """
+        Parameters
```

Comment:
return typehint missing :D oh boy, it would be cool to have some CI pipeline stage that does typechecks and so on, because now im getting nitpicky